data/brands/shop/supermarket.json

```diff
@@ -1920,13 +1920,15 @@
       }
     },
     {
-      "displayName": "denn's Biomarkt",
+      "displayName": "Denns BioMarkt",
       "id": "dennsbiomarkt-6367d0",
       "locationSet": {"include": ["at", "de"]},
       "tags": {
-        "brand": "denn's Biomarkt",
+        "brand": "Denns BioMarkt",
         "brand:wikidata": "Q48883773",
-        "name": "denn's Biomarkt",
+        "name": "Denns BioMarkt",
+        "operator": "Dennree GmbH",
```

Comment:
According to [Wikipedia](https://de.wikipedia.org/wiki/Denns_Biomarkt) (translated):
> Denns Biomarkt is an organic supermarket chain operated by the German Dennree GmbH.